lib/cylc/review.py

```diff
@@ -836,6 +839,19 @@ def _get_suite_logs_info(self, user, suite):
 
         return data
 
+    @staticmethod
+    def is_cylc8(user_suite_dir):
```

Comment:
Nice. Since it's a staticmethod couple of unit tests might be nice. You can probably mock the results of os.path.exists so that you don't have to touch the file system.